Engine/Module/ModuleScriptManager.cpp

```diff
@@ -234,11 +244,6 @@ void ModuleScriptManager::ReloadDLL()
 
 }
 
-bool ModuleScriptManager::CopyPDB(const char* source_file, const char* destination_file, bool overwrite_existing)
-{
-	return CopyFile(source_file, destination_file, !overwrite_existing);
-}
```

Comment:
I already knew in the file system was a function to copy files, but I did this because that function failed sometimes trying to manage PDB files. And this didn't, this refactor can manage properly the PDB's now?

Reply:
I tested a bit and it did, but I will prefer more testing to confirm the correct functionallity of the PDB update. Feel free to test it!